fdbserver/workloads/ValidateStorage.actor.cpp

```diff
@@ -85,11 +87,29 @@ struct ValidateStorage : TestWorkload {
 
 		loop {
 			try {
-				UID auditId = wait(auditStorage(cx->getConnectionRecord(), allKeys, AuditType::ValidateHA));
+				UID auditId_ =
+				    wait(auditStorage(cx->getConnectionRecord(), AuditType::ValidateHA, allKeys, /*async=*/true));
+				auditId = auditId_;
 				TraceEvent("TestValidateEnd").detail("AuditID", auditId);
 				break;
 			} catch (Error& e) {
-				TraceEvent("AuditStorageError").errorUnsuppressed(e);
+				TraceEvent("StartAuditStorageError").errorUnsuppressed(e);
```

Comment:
should cancellation error be caught here? And similar in the below try catch block

Reply:
Done.